### ISSUE TITLE
[Bug] Torus artifacts

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Skywanderers#1673]
### Short Description of the issue:
Sadly it looks like the issue from #1644 has resurfaced in a different form.
![Skywanderers Artifacts](https://user-images.githubusercontent.com/27857711/78085783-d1895500-73b3-11ea-91d5-6d7b1d7740ca.png)
These ones look much more similar to the stars in the new skybox, they're blue with a white center rather than the black squares we saw previously

### Steps to reproduce the issue:
Seems to be under the same conditions as before, along the same line on the torus seen when flying around normally

Hope these are less of a pain to figure out than the last time!

### Environment
- Skywanderers version: 20w13b
- Graphics Card: RX 580

### Additional information:
[output_log.txt](https://github.com/tsunamayo/Skywanderers/files/4412046/output_log.txt)

**Comment by tsunamayo** [comment]
Yes, and I saw some other on the terrain... But they did not impacted the blur. I keep you posted...
